client.go

```diff
@@ -665,7 +674,7 @@ func (r *exportReader) Read(p []byte) (n int, err error) {
 			r.frame.index.Name(), r.frame.Name(), r.currentSlice, r.view)
 		_, r.body, err = client.httpRequest("GET", path, nil, headers, errorCheckedResponse)
 		if err != nil {
-			return 0, err
+			return
```

Comment:
i guess my comment was more that the naked return can be confusing, especially in a longer function: https://tour.golang.org/basics/7